install/linux/usr/bin/odemis-select-mic-start

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+: '
+This script is used when there are multiple modular configurations available for METEOR.
+It is recommended to combine it with the exec= command of the Odemis.desktop file.
+However it could be used stand-alone if it is required.
+The syntax to run this script -> odemis-select-mic-start [search_dir] [search_pat].
+The script will search for configuration (.yaml) files with the used search directory
+and pattern. Once files are found microscope names are extracted and placed in a list.
+This list is presented to the user in the form of a (zenity) selection dialog window.
+After selection of the microscope configuration the right yaml file will be used to start Odemis.
+'
+# extract the arguments which were used to start this script
+args=( "$@" )
+
+for arg in ${args[@]}; do
+    if test "$arg" == "--help"; then
+        # Display Help
+        echo
+        echo "Syntax: odemis-select-mic-start [search_dir] [search_pat]"
+        echo "arguments:"
+        echo "search_dir     The directory to search for finding the main configuration files."
+        echo "search_pat     The search pattern to use to filter out specific file names."
+        echo
+        exit 0
+    fi
+done
+
+# check if exactly 2 arguments are used, any other combination is invalid
+if test "${#args[@]}" != "2"; then
+        echo "Wrong number of arguments, expected 2, got ${#args[@]}"
+        echo "Syntax is: odemis-select-mic-start [search_dir] [search_pat]"
+        exit 0
+fi
+
+# find all the files matching the search pattern and store them in an array
+files=("$(find ${args[0]} -name ${args[1]} -exec basename {} \;)")
```

Comment:
What I had in mind was to just pass directly the names of the yaml files as argument (full path). That would avoid fixing the search method inside the script. Also typically, I imagine you could just call the script as `odemis-select-mic-start /usr/share/odemis/*.yaml`, which is even simpler than the current way.

Or is there an advantage into explicitly passing the directory and search pattern?